### PR TITLE
Footer Section

### PR DESCRIPTION
**Pull request by kirtanchandak**
In this PR, I've developed the footer section!

### CODE REVIEW

File: website/src/components/footer/Footer.jsx [hunk 1]
```diff
@@ -14,16 +14,82 @@ See the License for the specific language governing permissions and
...
-        <div className='footer justify--content'>
-            <span>Copyright © 2023</span>
+  const icons = [
+    { url: "https://github.com/vitessio/arewefastyet", icon: AiFillGithub },
+    { url: "https://vitess.io/slack", icon: AiFillSlackCircle },
+    { url: "https://twitter.com/vitessio", icon: AiFillTwitterCircle },
+    {
+      url: "https://stackoverflow.com/search?q=vitess",
+      icon: BsStackOverflow,
+    },
```
Comment (frouioui) [inline website/src/components/footer/Footer.jsx:36]: Let's inline these fours lines too, just like we did in lines `30`, `31` and `32`

**Comment by frouioui** [comment]
Moreover, it would be great if the footer was the same width as the header and the other components of the page. With this change it feels like the footer is inside an even smaller container.

**Comment by kirtanchandak** [comment]
> Moreover, it would be great if the footer was the same width as the header and the other components of the page. With this change it feels like the footer is inside an even smaller container.

```
max-width: 100vw;
```
would also work, the current footer also has this

File: website/src/components/footer/Footer.jsx (hunk 1 above)
Comment (kirtanchandak) [reply website/src/components/footer/Footer.jsx:36]: okay

**Review by frouioui — APPROVED** [review]
LGTM @kirtanchandak thank you for this contribution!

**Pull request merged**